foreman/requirements.txt

```diff
@@ -6,6 +6,7 @@
 #
 amqp==2.1.4               # via kombu
 billiard==3.5.0.2         # via celery
+boto==2.46.1
```

Comment:
Interesting, it looks like support for boto3 will be available pretty soon though (https://github.com/celery/kombu/issues/592). Maybe in the next celery release!